scan-manager/docs/setup_artifacts/deployment/config.yml

```diff
@@ -1,11 +1,13 @@
-scikit_0.23:
-  deployment: scikit_0.23_deployment.yml
+python:
+  deployment: python_deployment.yml
```

Comment:
I'm replacing the `scikit_0.23` reference with `python`, and adding the current set of base images. We will need to update this when the release time comes (or figure out some way to automate this).

Follow up action would be to update the copy on our Certifai Dev to have `python`